docs/features/kubernetes/configuration.md

```diff
@@ -441,6 +476,14 @@ rules:
       - get
       - list
       - watch
+  # Required for fetching pod metrics
```

Comment:
This comment is a bit out of place with the rest of the example.

Reply:
Yes 😄

Reply:
Updated, thank you!